openc3/lib/openc3/models/cvt_model.rb

```diff
@@ -133,11 +144,23 @@ def self.get_tlm_values(items, stale_time: 30, scope: $openc3_scope)
 
     # Override a current value table item such that it always returns the same value
     # for the given type
-    def self.override(target_name, packet_name, item_name, value, type:, scope: $openc3_scope)
-      if VALUE_TYPES.include?(type)
-        @overrides["#{target_name}__#{packet_name}__#{item_name}__#{type}"] = value
+    def self.override(target_name, packet_name, item_name, value, type: :ALL, scope: $openc3_scope)
+      if type == :ALL
+        @overrides["#{target_name}__#{packet_name}__#{item_name}__RAW"] = value
```

Comment:
We can't store state in instance variables.... That will only work in a single process.   Need to store overrides in Redis.